### ISSUE TITLE
DOI external URLs not working

### ISSUE THREAD

**Issue by cmungall** [biocaddie/prototype_issues#286]
E.g.
https://datamed.org/display-item.php?repository=0028&id=581615629e761622d43cf591&query=CIAO1

![image](https://user-images.githubusercontent.com/50745/29039517-6fd0a51e-7b60-11e7-993d-fc21be6a6ed6.png)

The doi link goes to https://www.ncbi.nlm.nih.gov/pubmed/17994087%3Cbr%3Edoi:10.1038/nature06341

which returns an error from pubmed

**Comment by jmcmurry** [comment]
#prefixcommons

**Comment by ianfore** [comment]
There are two issues here
First a UI issue - there's an html <br> embedded in the URL which is separating the pubmed URL and the DOI
The first link is valid
https://www.ncbi.nlm.nih.gov/pubmed/17994087
That would be sufficient for user purposes

The DOI could be separated out - but as written it won't resolve. This would be form required as a URL relying on doi.org to resolve the doi.
https://doi.org/10.1038/nature06341
Datamed could do that expansion - but Julie's cryptic comment suggests prefixcommons might be the solution - but we need more details - Julie?

**Comment by ianfore** [comment]
Tried to understand how pervasive this is. It affected any of the UniProt records that I looked at - but only those that had publication ids.

I checked to see whether the problem is in the source data we're getting from UniProt
Looking at this [example](https://github.com/biocaddie/data-pipeline/blob/master/SampleData/uniprot_swissprot/swissprot_record_3.json) it seems that the two ids are present as distinct elements - so the badly formed URL must be occurring in the UI.
`        
"dbReference": [
          {
            "@id": "16912294",
            "@type": "PubMed"
          },
          {
            "@id": "10.1128/JVI.00464-06",
            "@type": "DOI"
          }
        ]
`

Seeing this throws some light on the URL issue. The fact that only the pubmed id (i.e. not the URL) is present in the imported data suggests that the UI code must be taking on the task of forming the pubmed URL. So it could also do so for DOIs - or use prefixcommons.
